README.md

```diff
@@ -51,11 +51,13 @@ $ jiraprinter --help
 
   Options:
 
-    -h, --help               output usage information
-    -V, --version            output the version number
-    -u, --user [username]    The JIRA username ($JIRA_USER)
-    --password [password]    The JIRA password ($JIRA_PASS)
-    -h, --host [host]        The JIRA hostname ($JIRA_HOST)
+    -h, --help                                  output usage information
+    -V, --version                               output the version number
+    -u, --user [username]                       The JIRA username ($JIRA_USER)
+    --password [password]                       The JIRA password ($JIRA_PASS)
+    -h, --host [host]                           The JIRA hostname ($JIRA_HOST)
+    -q, --printQR [printQR]                     Whether to include an issue link QR code in the print out ($PRINT_QR) - defaults to true
+    -c --customEstimation [customEstimation]    The name of a custom JIRA field to use in place of the built-in estimation field ($CUSTOM_ESTIMATION).
```

Comment:
I still think I'd like this to be more generic so that we can support other custom field names later - maybe something like --field-name estimation=foo.bar or something...

Also note that we don't _need_ a short-form for this sort of thing.